frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view.js

```diff
@@ -260,37 +258,6 @@ class TreeHierarchyView extends React.PureComponent<Props, State> {
     this.props.onDeleteGroup(groupId);
   }
 
-  handleDropdownClick = (params: { domEvent: SyntheticMouseEvent<*>, key: string }) => {
-    const { domEvent, key } = params;
-    // $FlowIssue[prop-missing] .dataset is unknown to flow
-    const groupId = parseInt(domEvent.target.dataset.groupId, 10);
-    if (key === "create") {
-      this.createGroup(groupId);
-    } else if (key === "delete") {
-      this.deleteGroup(groupId);
-    } else if (key === "collapseSubgroups") {
-      this.setExpansionOfAllSubgroupsTo(groupId, false);
-    } else if (key === "expandSubgroups") {
-      this.setExpansionOfAllSubgroupsTo(groupId, true);
-    }
-  };
-
-  handleTreeDropdownMenuVisibility = (treeId: number, isVisible: boolean) => {
-    if (isVisible) {
-      this.setState({ activeTreeDropdownId: treeId });
-      return;
-    }
-    this.setState({ activeTreeDropdownId: null });
-  };
-
-  toggleTreeDropdownMenuVisibility = (treeId: number) => {
```

Comment:
This function seemed to be a duplicate of `handleTreeDropdownMenuVisibility`. I could not find any usages.